### PR TITLE
[CLOUDTRUST-1423] Support paging for GetUsers

### PR DESCRIPTION
**Pull request by fperot74**
This PR is related to:
* cloudtrust/keycloak-rest-api-extensions#14
* cloudtrust/keycloak-client#14

⚠️ Build can't be successful until cloudtrust/keycloak-client#14 is merged

### CODE REVIEW

File: api/events/swagger-api_events.yaml [hunk 1]
```diff
@@ -66,14 +66,9 @@ paths:
                 type: object
                 properties:
                   events:
-                    type: object
```
Comment (Fratt) [inline api/events/swagger-api_events.yaml]: Why did you change this?

Comment (fperot74) [reply api/events/swagger-api_events.yaml]: There were two levels with a `count` field

Comment (Fratt) [reply api/events/swagger-api_events.yaml]: My bad
